README.md

```diff
@@ -0,0 +1,186 @@
+# Hazelcast Discovery Plugin for Kubernetes
+
+This repository contains a plugin which provides the automatic Hazelcast member discovery in the Kubernetes environment.
```

Comment:
I suggest to explain why someone should use this plugin. We can point out this plugin is already available in hazelcast-docker image or in hazelcast helm chart. 
Main usage of this plugin is to embed hazelcast into application docker image so that can be pointed out. Otherwise, Users need to be directed to helm chart or base docker image.

Reply:
I see Embedded section below is explaining the usage. Lets just move up embedded section is so users can see the usage pattern in the first place.

Reply:
done